### ISSUE TITLE
Online only content toggle should have only one field label

### ISSUE THREAD

**Issue by Seanwarren77** [pulibrary/pulfalight#1488]
### Steps to recreate
https://wave.webaim.org/report#/https://findingaids.princeton.edu/catalog/WC064_c1

### URL
https://findingaids.princeton.edu/catalog/WC064_c1

### Issues
WAVE accessibility report rose errors due to multiple field labels. 

### Severity
High Severity

### WCAG Success Criterion
https://www.w3.org/TR/WCAG22/#labels-or-instructions

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Users of assistive technologies may not be given the appropriate label. 

### Recommended fix
Remove one of the labels for `tocOnlineToggle`

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
